Assets/HoloToolkit/Utilities/Scripts/Solvers/AdditionalSolverHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public class AdditionalSolverHandler : SolverHandler
```

Comment:
What's the purpose of this one that isn't supported by the regular SolverHandler?

Reply:
Plus, that's an odd name. If we need a second SolverHandler, I would prefer that it have a name that implies its purpose.

Reply:
Normal SolverHandlers run SolverUpdate() during the LateUpdate() on all the solvers on the game object. AdditionalSolverHandlers don't call SolverUpdate() at all, instead relying on the SolverUpdate() to update them. 

Similarly, SolverHandlers call SeekTrackedObject() on all solvers when they need to reattach. AdditionalSolverHandler only calls it on the relevant solver.

This does remind me that we should require a SolverHandler component, since this relies on a SolverHandler managing SolverUpdate().

Reply:
@davidkline-ms Any ideas for an alternative? SolverHandlerForSecondTransform? SolverHandlerForAdditionalTransform? SecondaryTrackedObjectSolverHandler?

Reply:
I like SecondaryTrackedObjectSolverHandler.